index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <title>Portfolio - Bright Kweku Ahiadeke</title>
+      <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header>
+      <nav class="header-nav wrap">
+        <h1 class="logo-text">Bright</h1>
```

Comment:
- [ ] [REQUIRED] I think it would be better if you replace ♻️ the `<h1>` element with a `<a>` element for semantic purposes and also in order for the logo to be clickable (as users used to on the internet 🌐)